app/assets/stylesheets/application.scss

```diff
@@ -21,12 +20,43 @@ $black: #000000;
 $red: #ff0000;
 $search-result: #008cba;
 
+body {
+  background-attachment: fixed;
+  background-image: image-url("bb_background_small.jpg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.top-bar {
+  // background-color: rgba(244, 162, 23, 0.66);
+}
+
+.right {
+  background-color: rgba(244, 162, 23, 0.66);
+  position: relative;
+  right: 0px;
+}
+
 .hidden-button {
  display: none;
 }
 
+.home-link {
+  color: rgb(255, 255, 255);
+}
+.main-area {
+  background:rgba(108, 108, 108, 0.5);
+  margin: 30px;
```

Comment:
Properties should be ordered background, -moz-border-radius, -webkit-border-radius, border-radius, margin, margin-top, padding